measure_accuracy.py

```diff
@@ -0,0 +1,24 @@
+import cv2
+from os import listdir
+import numpy as np
+
+path = "/Users/akshatmehta/Downloads/benchmark/"
+for i in listdir(path):
+
+    test = cv2.imread(path + i)
+    print(i[:-16])
+    correct = cv2.imread("/Users/akshatmehta/Downloads/binary/" + i[:-16] + ".jpg")
+
+    matches = 0
+
+    for i in range(len(test)):
+        for j in range(len(test[i])):
+            if test[i][j][0] == correct[i][j][0]:
+                matches += 1
+
+    accuracy = matches / 2073600
```

Comment:
What is 2073600?

Reply:
It's the number of pixels in the image - 1920*1080 = 2073600

Reply:
We should calculate this based on the image we're given, not hard code it.